src/engine/QueryPlanner.cpp

```diff
@@ -2535,8 +2535,12 @@ void QueryPlanner::GraphPatternPlanner::visitTransitivePath(
     right.value_ = getSideValue(arg._right);
     size_t min = arg._min;
     size_t max = arg._max;
+    if (planner_.activeGraphVariable_.has_value()) {
+      AD_THROW("Property paths with graph variables are not supported");
```

Comment:
not AD_THROW but simply throw
(it is not a bug, but an unsupported feature)
and it should read `Property paths inside a GRAPH clause with a graph variable are...`.